posts/2021/04/spot-the-differences.md

```diff
@@ -0,0 +1,213 @@
+<!--
+.. title: Spot the differences: what is new in Spyder 5?
+.. slug: spot-the-diffenrences
+.. date: 2021-04-16 08:00:00 UTC-06:00
+.. author: Isabela Presedo-Floyd and Juanita Gomez
+.. tags: Spyder, UX/UI, release
+.. category:
+.. link:
+.. description:
+.. type: text
+-->
+
+![Spyder 5 versus Spyder 4](/images/spyder5-header.png)
+
+*This blog post is a conversation. Portions lead by Juanita Gomez are marked 
+as **Juanita**, and those lead by Isabela Presedo-Floyd are marked as 
+**Isabela**.*
+
```

Comment:
Unless folks have context this might be a bit confusing I would open with something like: 

In case you missed it Spyder 5 was released at the beginning of April and link to the repo

Reply:
Even if this is also a few lines down - you always want to start a blog with a hook

Reply:
Combining this with one the other lingering comments, I think the opening could be like

`In case you missed it Spyder 5 was released at the beginning of April! This blog post is a conversation attempting to document the long and complex process of improving Spyder's UI with this release. Portions lead by Juanita Gomez are marked  as **Juanita**, and those lead by Isabela Presedo-Floyd are marked as **Isabela**.`

and then immediately after go into the existing intro

` **[Juanita]** Spyder was created 10 years ago…`

Does that seem like it could resolve the comments and flow better?

Reply:
I will implement this as a separate commit once we agree on the other suggestions since it's kind of long.